src/lib/components/viewer/PDF.svelte

```diff
@@ -61,7 +61,7 @@
         window.pdf = p;
       })
       .catch((e) => {
-        console.error(e);
+        console.warn(e);
```

Comment:
Rendering PDFs is on the critical path—I think it's fair to treat these as elevated.